runtime/j9vm/java11vmi.c

```diff
@@ -550,10 +560,17 @@ getJ9Module(J9VMThread * currentThread, jobject module)
 }
 
 static BOOLEAN
-isModuleJavaBase(j9object_t moduleName)
+isModuleJavaBase(J9VMThread * currentThread, j9object_t moduleName)
 {
-	/** @todo compare against string 'java.base' */
-	return FALSE;
+	PORT_ACCESS_FROM_VMC(currentThread);
```

Comment:
can this comparison be done with `compareStringToUTF8` instead?

Reply:
Good plan.  Thanks.